### DISCUSSION TITLE
How to add different reviewers depending on updated code?

### DISCUSSION THREAD

**Discussion by PhilipDukhov** [renovatebot/renovate#21899] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I have a monorepo, it contains code for different platforms.

I'd like to be able to request for a review for a user/users depending on updated platform(gradle/npm/etc) or by folder where changes are applied.

### What you tried so far.

I was looking for something relevant in [documentation](https://docs.renovatebot.com/configuration-options/) but haven't found it.

**Comment by rarkins** [comment, marked as answer]
Use codeowners and GitHub will do it for you. Otherwise you can use something like matchPaths+assignees in packageRules
